sphinx/dev-tools.rst

```diff
@@ -84,7 +84,11 @@ Testing
 
 Run tests and generate coverage report with :github-source:`dev-tools/test.sh`::
 
-    ./dev-tools/test.sh
+    ./dev-tools/test.sh [test_path]
+
+**Arguments:**
+
+* ``test_path``: Run only tests from the given path
```

Comment:
```suggestion
    ./dev-tools/test.sh [TEST_PATH]

**Arguments:**

* ``TEST_PATH``: Run only tests in ``TEST_PATH``
```
I guess for "variable" arguments, there are two conventions, either angular brackets `<var>` or upper case `VAR`, to differentiate them from the literal argument `var` (see e.g. http://docopt.org/)...

Reply:
Probably [<test_path>] would be better actually...
I only see the use of uppercase for the argument of option 
`--input=ARG`

Reply:
Well, I think both are fine, e.g. the Django CLI also uses upper case:
```
pipenv run integreat-cms-cli shell -h
usage: integreat-cms-cli shell [-h] [--no-startup] [-i {ipython,bpython,python}] [-c COMMAND]
                               [--version] [-v {0,1,2,3}] [--settings SETTINGS]
                               [--pythonpath PYTHONPATH] [--traceback] [--no-color]
                               [--force-color]
```
Choose whichever you prefer :smile:

Reply:
`-c COMMAND`
in your example it is also the argument of an option. but our TEST_PATH is a positional argument :)